src/common/meson.build

```diff
@@ -1,6 +1,35 @@
+rc_misc_c = files([
+  'rc-misc.c',
```

Comment:
since these files are in common/ now, do they still really need the `rc-` prefix on the filename ?